src/main/java/liquibase/ext/neo4j/change/ColumnMapper.java

```diff
@@ -0,0 +1,111 @@
+package liquibase.ext.neo4j.change;
+
+import liquibase.change.ColumnConfig;
+import liquibase.change.ColumnConfig.ValueNumeric;
+import liquibase.change.core.LoadDataChange.LOAD_DATA_TYPE;
+import liquibase.change.core.LoadDataColumnConfig;
+import liquibase.ext.neo4j.exception.UnsupportedLoadDataTypeException;
+import liquibase.statement.DatabaseFunction;
+
+import java.time.ZonedDateTime;
+import java.time.temporal.Temporal;
+import java.util.Base64;
+
+class ColumnMapper {
+
+    public static Object mapValue(ColumnConfig column) {
+        Object value = column.getValueObject();
+        if (value instanceof ValueNumeric) {
+            return mapNumericValue((ValueNumeric) value);
+        }
+        String type = column.getType();
+        if (type == null) {
+            if (column.getValueSequenceNext() != null || column.getValueSequenceCurrent() != null) {
+                throw new IllegalArgumentException(
+                        "sequence values are currently not supported by the Neo4j plugin"
+                );
+            }
+            if (column.getValueComputed() != null) {
+                throw new IllegalArgumentException(
+                        "computed values are currently not supported by the Neo4j plugin"
+                );
+            }
+            return value;
+        }
+        switch (type) {
+            case "date":
+                return mapTemporalValue(column.getValueObject());
+            case "blob":
+                return mapBlobValue(column);
+            case "clob":
+                return mapClobValue(column);
+        }
```

Comment:
Probably good to have a default case so nothing falls through.

Reply:
fixed in ceaf54a51e27cf1d952b4a61a2cd45ef1b7cf7ad